.github/CODEOWNERS

```diff
@@ -1,86 +1,86 @@
+/src/material/snack-bar/**                         @andrewseguin @crisbeto
 /src/material/sort/**                              @andrewseguin
 /src/material/stepper/**                           @mmalerba
 /src/material/table/**                             @andrewseguin
 /src/material/tabs/**                              @andrewseguin
-/src/material/testing/**                           @jelbourn
+/src/material/testing/**                           @andrewseguin
 /src/material/toolbar/**                           @devversion
 /src/material/tooltip/**                           @andrewseguin
 /src/material/tree/**                              @jelbourn @andrewseguin
 
 # Angular Material core
-/src/material/core/*                               @jelbourn
+/src/material/core/*                               @andrewseguin
 /src/material/core/testing/**                      @crisbeto
-/src/material/core/animation/**                    @jelbourn
-/src/material/core/color/**                        @jelbourn @devversion
-/src/material/core/common-behaviors/**             @jelbourn @devversion
+/src/material/core/animation/**                    @andrewseguin
+/src/material/core/color/**                        @andrewseguin @devversion
+/src/material/core/common-behaviors/**             @andrewseguin @devversion
 /src/material/core/datetime/**                     @mmalerba
 /src/material/core/density/**                      @devversion
 /src/material/core/error/**                        @crisbeto @mmalerba
 /src/material/core/focus-indicators/**             @jelbourn @zelliott
-/src/material/core/gestures/**                     @jelbourn
+/src/material/core/gestures/**                     @andrewseguin
 /src/material/core/label/**                        @mmalerba
-/src/material/core/line/**                         @jelbourn
+/src/material/core/line/**                         @andrewseguin
 /src/material/core/option/**                       @crisbeto
 /src/material/core/placeholder/**                  @mmalerba
 /src/material/core/ripple/**                       @devversion
-/src/material/core/selection/**                    @jelbourn
-/src/material/core/selection/pseudo*/**            @crisbeto @jelbourn
-/src/material/core/style/**                        @jelbourn
-/src/material/core/theming/**                      @jelbourn
+/src/material/core/selection/**                    @andrewseguin
+/src/material/core/selection/pseudo*/**            @crisbeto @andrewseguin
+/src/material/core/style/**                        @andrewseguin
+/src/material/core/theming/**                      @andrewseguin @jelbourn
 /src/material/core/typography/**                   @crisbeto
-/src/material/core/util/**                         @jelbourn
+/src/material/core/util/**                         @andrewseguin
 
 # Miscellaneous components
 /src/google-maps/**                                @mbehrlich
 /src/youtube-player/**                             @nathantate
```

Comment:
Should we have more people on `google-maps` and `youtube-player`?

Reply:
Added you